src/containers/WelcomePage/components/Pagination.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { ChevronLeft, ChevronRight } from "@ndla/icons/common";
+import {
+  Button,
+  PaginationContext,
+  PaginationEllipsis,
+  PaginationItem,
+  PaginationNextTrigger,
+  PaginationPrevTrigger,
+  PaginationRoot,
+  Text,
+} from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
+import { usePaginationTranslations } from "@ndla/ui";
+
+const StyledPaginationRoot = styled(PaginationRoot, {
+  base: {
+    flexWrap: "wrap",
+  },
+});
+
+interface Props {
+  page: number | undefined;
+  onPageChange: (details: { page: number; pageSize: number }) => void;
+  count: number | undefined;
+  pageSize: number | undefined;
+}
+
+const Pagination = ({ page = 1, onPageChange, count = 0, pageSize }: Props) => {
+  const translations = usePaginationTranslations();
+  return (
+    <StyledPaginationRoot
+      page={page}
+      onPageChange={onPageChange}
+      count={count}
+      pageSize={pageSize}
+      translations={translations}
```

Comment:
Lurer faktisk på om denne burde ha en aria-label. Er det ikke en nav?

Reply:
Det gjelder nok de andre stedene vi bruker pagination her også.

Reply:
Det er nok lurt. Se ndla-frontend for bruk!

Reply:
Har du forslag til hva det skal være? Ressurssidenavigering? skal det være forskjellig på de ulike tabellene?